### DISCUSSION TITLE
macOS: `cmd+k=clear_screen` keybinding is not working

### DISCUSSION THREAD

**Discussion by unexge** [ghostty-org/ghostty#3382] [Q&A] [answered]
I'm trying to assign `cmd + k` keybinding to clear the screen as I used that behavior from other terminals, but it's not working for some reason.

I have the following configuration:
```bash
$ ghostty +list-keybinds | grep clear_screen
keybind = super+k=clear_screen
```

When I press `cmd + k` it just inputs `k` instead of clearing the screen.

https://github.com/user-attachments/assets/156165b3-6c47-41e3-87ec-26d3537610fb

The default clear screen action (`ctrl + l`) works fine.

Not sure if it's related but when I check the keyboard events, I can see release event for `ctrl + l ` but not for `cmd + k` (only for `cmd`):
<img width="1086" alt="Screenshot 2024-12-27 at 11 58 20" src="https://github.com/user-attachments/assets/47c1ad13-b569-43e3-bcd8-3f78798d024d" />

**Comment by yochem** [comment, marked as answer]
This might be because of Zelij. Could you try it without running Zelij? I have the same issue with tmux. The fix is to send ctrl + l instead:

```
keybind = command+k=text:\x0c
```

I forgot I also had this remapping in my iTerm config 😅

**Reply by unexge** [reply]
Yes that was indeed the case! It works without Zellij and changing keybind as you suggested also fixes the problem with Zellij. Thanks!

**Comment by patrykk21** [comment]
I don't have zellij and I have the same issue

**Reply by yochem** [reply]
When you open the inspector, does it show that it receives cmd+k?

**Reply by hrdkbhatnagar** [reply]
I am having the same issue. I dont have zellij and pressing cmd+k just outputs the character k in the terminal. 

The inspector shows that it receives cmd+k 
<img width="940" alt="image" src="https://github.com/user-attachments/assets/18bf6512-ebdc-47cd-b8bd-4df118a4c9af" />

**Reply by rhodes-b** [reply]
> I have the same issue with tmux

Looks like you are also using tmux

**Comment by qu8n** [comment]
If you use tmux and the above solutions don't work, try:
```
keybind = cmd+k=text:clear\r
```

**Reply by rijieli** [reply]
Thanks, works fine on `zsh`.
